src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.stars__star:nth-child(-n+4) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
Rewrite this with checklist
<img width="360" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/cd05e92c-f209-4903-b52a-c326be8c1c38">